examples/00_basic/example_01_simple_structural_solve.py

```diff
@@ -0,0 +1,212 @@
+""".. _ref_example_01_simple_structural_solve:
+
+Static structural analysis
+--------------------------
+
+Using supplied files, this example shows how to insert a static structural
+analysis into a new Mechanical session and execute a sequence of Python scripting
+commands that define and solve the analysis. Deformation results are then reported.
```

Comment:
```suggestion
commands that define and solve the analysis. The example then shows how
to report deformation results.
```
Use active rather than passive voice.